### DISCUSSION TITLE
Job board

### DISCUSSION THREAD

**Discussion by peterpetrov** [joomengine/Joomla-Component-Builder#791] [Ideas]
Hello guys,

It was some time ago i have interacted with the community. I hope all goes well with everyone and you are safe and healthy.

Some time ago we were discussing a topic about passing clients to each other in case there was high workload or any other circumstances are in play. I would like to raise that question again. I have 1 request that includes some custom development and i know here are many people more than capable for doing the job. I have also 1 client that at some point will want to pass to someone to support them as i am going in completely another direction away from Joomla and need all of my free time. 

So my question is - how do you think is best to do that within the community? Since we are not using the Google forum should it be here to share the websites and clients or we can have another tool to do it?

I would be happy with any answer.

Regards,
Peter

**Comment by EvDoorne** [comment]
I think it's up to you Peter how to handle this. You have a group of skilled developers here that could be of service.

**Comment by Llewellynvdm** [comment]
I can add a new category here, and don't mind this kind of collaboration here in discussions... I think it is ideal to have as much of this in one place, that is why we moved here.

I am just not sure what to call this new category... there is already a [show and tell](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/show-and-tell) category to share your work. So we can add maybe [Opportunities](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/opportunities) or [Jobs](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/jobs)

Which will be more fitting?

- [x] [Opportunities](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/opportunities)
- [ ] [Jobs](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/jobs)

**Reply by EvDoorne** [reply]
Opportunies

**Reply by ohrionmartin** [reply]
> I can add a new category here, and don't mind this kind of collaboration here in discussions... I think it is ideal to have as much of this in one place, that is why we moved here.
> 
> I am just not sure what to call this new category... there is already a [show and tell](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/show-and-tell) category to share your work. So we can add maybe [Opportunities](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/opportunities) or [Jobs](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/jobs)
> 
> Which will be more fitting?
> 
>     * [ ]  [Opportunities](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/opportunities)
> 
>     * [ ]  [Jobs](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/jobs)
 

- [x] [Opportunities](https://github.com/vdm-io/Joomla-Component-Builder/discussions/categories/opportunities)

**Reply by peterpetrov** [reply]
Opportunities sounds good.

**Reply by Llewellynvdm** [reply]
Done!
